controllers/villains.js

```diff
@@ -0,0 +1,35 @@
+const { villains } = require('../models/index')
+
+const getAllVillains = async (req, res) => {
+  const allVillains = await villains.findAll()
+
+  return res.status(200).send(allVillains)
+}
+
+const getBySlug = async (req, res) => {
+  const searchedSlug = req.params.slug
+
+  const villain = await villains.findOne({
+    where: { slug: searchedSlug }
```

Comment:
you can use the attribute property which will only display certain attributes on the page.

const villain = await villains.findOne({ where: { slug: searchedSlug }, attributes: ['name', 'movie', 'slug'], })